Swift/BRCrypto/common/BRCryptoKey.swift

```diff
@@ -192,93 +148,45 @@ public final class Key {
 
         var secret = secret
         defer { secret = UInt256() }
+        // Serialization - Public Key
 
         BRKeySetSecret (&core, &secret, 1)
```

Comment:
Could this be moved to C as well? Selfish goal is to have no non-crypto calls in Java (and Swift).

Reply:
Yes.